.npmignore

```diff
@@ -1,12 +0,0 @@
-*.DS_Store
```

Comment:
Wondering why `.npmignore` is removed?

Reply:
The only file that needs to be published is index.js so I put that in package.json

I have documented it in the [commit](https://github.com/Ultimaker/eslint-config/pull/9/commits/690ee8133ab335c8e62cad675d2c8c83445aebb6)